tests/draft-next/dynamicRef.json

```diff
@@ -205,47 +205,75 @@
     {
         "description": "multiple dynamic paths to the $dynamicRef keyword",
         "schema": {
-            "$schema": "https://json-schema.org/draft/next/schema",
+            "$schema": "https://json-schema.org/draft/2020-12/schema",
             "$id": "https://test.json-schema.org/dynamic-ref-with-multiple-paths/main",
+            "propertyDependencies": {
+              "numbers": { "$ref": "numberList" },
+              "strings": { "$ref": "stringList" }
+            },
```

Comment:
I don't think this is right.  It should be a dictionary of dictionaries of strings.  Maybe

```json
"propertyDependencies": {
  "kindOfList": {
    "numbers": { "$ref": "numberList" },
    "strings": { "$ref": "stringList" }
  }
}
```

Reply:
Thank you. I did test it, but that doesn't help if I don't commit the version I tested :man_facepalming:.